Components/llsp.c

```diff
@@ -26,7 +26,7 @@ struct matrix {
 struct llsp_s {
 	size_t        metrics;   // metrics count
 	double       *data;      // pointer to the malloc'ed data block, matrix is transposed
-	struct matrix full;      // pointers to the matrix in its original form with all columns
+	struct matrix full;      // pointers to the matrix in its original form with all columns 指向具有所有列的原始形式的矩阵的指针
```

Comment:
Do not put comments in Chinese in a repo that is supposed to be widely accessible.